cfgov/filing_instruction_guide/jinja2/filing_instruction_guide/layout-fig.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'layout-1-3.html' %}
+
+{% set version_statuses = {
+    'current': '',
+    'old': 'Previous version',
+    'archived': 'Archived'
+} %}
+{% set version_status = version_statuses[page.version_status] %}
+{% set effective_start_date = page.effective_start_date.strftime('%B %d, %Y') if page.effective_start_date %}
+{% set effective_end_date = page.effective_end_date.strftime('%B %d, %Y') if page.effective_end_date %}
+{% set version_message = 'This version has been archived.' if version_status == 'Archived' else 'This version is not the current FIG.' %}
+{% set date_message = 'You are viewing a previous version of the FIG.' %}
+{% if effective_start_date and effective_end_date %}
+    {% set date_message = 'You are viewing a previous version of the FIG, effective ' + effective_start_date + ' to ' + effective_end_date + '.' %}
+{% endif %}
+{% set banner_modification = ' o-banner__dark' if version_status == 'Archived' else ''  %}
+
+{% block banner_top %}
+    {% if version_status %}
+        {% import 'molecules/notification.html' as notification with context %}
+        <div class="o-banner{{ banner_modification }}">
+            <div class="wrapper
+                        wrapper__match-content">
+                {{ notification.render(
+                    'warning',
+                    true,
+                    version_message,
+                    date_message,
+                    [ {
+                        'text': 'View all versions of the FIG.',
+                        'url': '#'
```

Comment:
I think (?) the index of all versions will probably be one level up from the FIG page itself, but I'm not sure. In any case, we should put a TODO around this and create a ticket so we remember to update it.

```suggestion
                        'url': '../'
```

Reply:
Another thought: maybe this should be defined in the CMS? A field in wagtail that lets the user input the URL of the index/history page in the event we don't list all the versions on the FIG landing page.